conditional-admin-action/README.md.liquid

```diff
@@ -0,0 +1,11 @@
+# Conditional admin action extension group
+
+## Action extension
+
+Admin action extensions enable app developers to build custom functionality into the context of Shopify Admin. These extensions surface as launchable menu actions that open interactive modals at various extension targets and enhance the merchant experience. Developers can build the content of these extensions using Shopify's UI Extension components for Admin.
+
+Learn more about Admin action extensions in Shopify’s [developer documentation](https://shopify.dev/docs/apps/admin/admin-actions-and-blocks).
+
+## Admin data extensions to conditionally control action extension triggers
+
+Admin data extensions run in the browser like UI extensions but do not return any UI. Instead, an admin data extension returns specific data, as permitted by the contract with the extension point that it runs in. Admin action condition extensions are a specific type of admin data extension used to control the visibility of an admin action trigger in the "More actions" menu on a given resource page. They are expected to return a data object that determines the visibility of a related admin action extension. 
```

Comment:
```suggestion
Admin data extensions run in the browser like UI extensions but do not return any UI. Instead, an admin data extension returns specific data, as permitted by the contract with the extension point that it runs in. Admin action condition extensions are a specific type of admin data extension used to control the intractability of an admin action trigger in the "More actions" menu on a given resource page. They are expected to return a data object that determines the visibility of a related admin action extension. 
```

Reply:
I'm not if the hiding vs greying out decision is final. @AaronSpringut did you finalize this with UX?

Reply:
Not yet, hopefully an update in 45 min.